src/automations/components/AutomationActionDescriptionSendNotification.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="automation-action-description-send-notification">
+    Send {{ indefiniteArticle() }} {{ blockDocument?.blockType.name.toLocaleLowerCase() ?? '' }} notification using
+    <BlockIconText :block-document-id="props.action.blockDocumentId" />
```

Comment:
This works as is but is kinda an anti pattern. Props are automatically available in the template by their names. So you don't need to access the `props` reactive object. 

```suggestion
    <BlockIconText :block-document-id="action.blockDocumentId" />
```